mgmt/rest/client/task.go

```diff
@@ -93,6 +93,9 @@ func (c *Client) CreateTask(s *Schedule, wf *wmap.WorkflowMap, name string, dead
 // interactive with Event and Done channels. An HTTP GET request retrieves tasks.
 // StreamedTaskEvent returns if it succeeds. Otherwise, an error is returned.
 func (c *Client) WatchTask(id string) *WatchTasksResult {
+	// during watch we don't want to have a timeout
+	c.http.Timeout = time.Duration(0)
+
```

Comment:
That is a good point. In the context of this fix I was thinking of the only user as snaptel which wouldn't be affected by this.